Gordon360/Controllers/MembershipsController.cs

```diff
@@ -44,12 +45,13 @@ public ActionResult<IEnumerable<MembershipView>> GetMemberships(string? involvem
             {
                 var authenticatedUserUsername = AuthUtils.GetUsername(User);
                 var viewerGroups = AuthUtils.GetGroups(User);
+                bool mp = myProf ?? false;
```

Comment:
it occurred to me that we could make this variable defaulted in the parameter list
```suggestion
```